db.js

```diff
@@ -0,0 +1,68 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost:27017/overview', { useNewUrlParser: true, useUnifiedTopology: true });
+
+const { Schema } = mongoose;
+
+const FeatureSchema = new Schema({
+  feature: String,
+  value: String,
+});
+
+const PhotoSchema = new Schema({
+  thumbnail_url: String,
+  url: String,
+});
```

Comment:
Why have this as a separate schema as opposed to just put this object inside the brackets on line 23?